Add Beam Faucet & RPC details (#1503)
ava-labs/avalanche-docs: docs/build/subnet/utility/avalanche-subnet-faucet.md

or [Subnet](learn/avalanche/subnets-overview.md) that is going live on the main network (Mainnet). For
example, [Fuji](learn/avalanche/fuji.md) network is the Testnet for Avalanche's Mainnet.

> Besides Fuji Testnet, the [Avalanche Faucet](https://core.app/tools/testnet-faucet/?subnet=c&token=c) 
> can be used to get free test tokens on testnet Subnets like: 

Reviewer: ```suggestion
Besides Fuji Testnet, the
[Avalanche Faucet](https://core.app/tools/testnet-faucet/?subnet=c&token=c) 
can be used to get free test tokens on testnet Subnets like: 
```

or [Subnet](learn/avalanche/subnets-overview.md) that is going live on the main network (Mainnet). For
example, [Fuji](learn/avalanche/fuji.md) network is the Testnet for Avalanche's Mainnet.

> Besides Fuji Testnet, [Avalanche Faucet](https://core.app/tools/testnet-faucet/?subnet=c&token=c) can be used
> to get free coins
> on these testnets Subnets like -

Reviewer: ```suggestion
Besides Fuji Testnet, the [Avalanche Faucet](https://core.app/tools/testnet-faucet/?subnet=c&token=c) 
can be used to get free test tokens on testnet Subnets like: 
```

> Besides Fuji Testnet, [Avalanche Faucet](https://faucet.avax.network/) can be used to get free coins

Reviewer: ```suggestion
Besides Fuji Testnet, [Avalanche Faucet](https://core.app/tools/testnet-faucet/?subnet=c&token=c) can be used
to get free coins
```

> - [WAGMI Testnet](https://faucet.avax.network?subnet=wagmi)
> - [DeFI Kingdoms Testnet](https://faucet.avax.network?subnet=dfk)

Reviewer: ```suggestion
- [WAGMI Testnet](https://core.app/tools/testnet-faucet/?subnet=wagmi)
- [DeFI Kingdoms Testnet](https://core.app/tools/testnet-faucet/?subnet=dfk)
- [Beam Testnet](https://core.app/tools/testnet-faucet/?subnet=beam&token=beam) and many more.
```

> - [Dexalot Testnet](https://faucet.avax.network?subnet=dexalot)
> - [Beam Testnet](https://core.app/tools/testnet-faucet/?subnet=beam&token=beam) and many more.

Reviewer: ```suggestion
```
